intel_extension_for_transformers/llm/runtime/graph/application/main_pybind.cpp

```diff
@@ -511,54 +520,71 @@ int Model::quant_model(const std::string& model_path, const std::string& out_pat
 }
 
 size_t Model::jblas_qpack(const int8_t* src_w, const float* src_scales, const int8_t* src_zps, void* dstpr,
-                          const quant_params_internal params, int nthread, int n, int k) {
-  using CompType = jblas::prologue::weight_comp::gemm_kblcok::PrologueBIDs;
-  using namespace ne_jblas;
-  auto cd = jblas::utils::parallel::CpuDevice::getInstance();
+                          const quant_params_internal params, int nthread, int N, int K) {
+  
+  auto constexpr ISA = jblas::gemm::ICoreRowNAvx512vnniKBlock<48, 4>::ISA;
```

Comment:
bound to single ISA?

Reply:
this function is used by gptq, I solved it in gptq branch.

Reply:
this pr https://github.com/intel/intel-extension-for-transformers/pull/791 ?

Reply:
not this, I will push new PR after this one has been merged.